src/packages/cli/src/bin.ts

```diff
@@ -175,9 +175,17 @@ async function main(): Promise<number> {
     return 1
   }
   console.log(result)
+
+  // Project hash is a SHA256 of the schemaPath
+  const projectHash = await getProjectHash()
+  // SHA256 of the cli path
+  const cliPathHash = getCLIPathHash()
+
   // check prisma for updates
   const checkResult = await checkpoint.check({
     product: 'prisma',
+    cli_path_hash: cliPathHash,
+    project_hash: projectHash,
```

Comment:
nit: also is a path hash, schema path to be precise

Reply:
That's true. 

I kept it that way so that consumers of checkpoint-client can decide what they use to identify a project. In the context of the checkpoint-client, I thought it'd be good to keep it projectHash as it could use a different value to identify a project. 

Nevertheless here it could be renamed to `schemaPathHash`

Reply:
Good point, depending on context (client vs. server) it indeed can be differentiated.